linera-views/src/backends/lru_caching.rs

```diff
@@ -346,38 +363,49 @@ where
     async fn new_test_config() -> Result<LruCachingConfig<K::Config>, K::Error> {
         let inner_config = K::new_test_config().await?;
         let cache_size = TEST_CACHE_SIZE;
+        let cache_key_absence = false;
         Ok(LruCachingConfig {
             inner_config,
             cache_size,
+            cache_key_absence,
         })
     }
 }
 
-fn new_lru_prefix_cache(cache_size: usize) -> Option<Arc<Mutex<LruPrefixCache>>> {
+fn new_lru_prefix_cache(
+    cache_size: usize,
+    cache_key_absence: bool,
+) -> Option<Arc<Mutex<LruPrefixCache>>> {
     if cache_size == 0 {
         None
     } else {
-        Some(Arc::new(Mutex::new(LruPrefixCache::new(cache_size))))
+        Some(Arc::new(Mutex::new(LruPrefixCache::new(
+            cache_size,
+            cache_key_absence,
+        ))))
     }
 }
 
 impl<K> LruCachingStore<K> {
     /// Creates a new key-value store that provides LRU caching at top of the given store.
-    pub fn new(store: K, cache_size: usize) -> Self {
-        let lru_read_values = new_lru_prefix_cache(cache_size);
+    pub fn new(store: K, cache_size: usize, cache_key_absence: bool) -> Self {
+        let lru_read_values = new_lru_prefix_cache(cache_size, cache_key_absence);
         Self {
             store,
             lru_read_values,
         }
     }
 
-    /// Gets the `cache_size`
-    pub fn cache_size(&self) -> usize {
+    /// Gets the `cache_config`
+    pub fn cache_config(&self) -> (usize, bool) {
```

Comment:
(I'm fine with this for now, but in the long run, if we keep the option, we should probably make this a struct rather than a tuple.)

Reply:
Ok, next PRs.